.teamcity/_self/projects/WPComTests.kt

```diff
@@ -301,6 +305,10 @@ fun jetpackAtomicDeploymentE2eBuildType( targetDevice: String, buildUuid: String
 			param("env.VIEWPORT_NAME", "$targetDevice")
 			param("env.JETPACK_TARGET", "wpcom-deployment")
 			param("env.TEST_ON_ATOMIC", "true")
+			// We run all the tests on all variations, and go through each variation sequentially.
+			// We can easily overwhlem the target Atomic site under test if we have too much parallelization.
+			// This number of works plays nicely with the expected load handling on these Atomic sites.
+			param("JEST_E2E_WORKERS", "8")
```

Comment:
I'm going to add a link to the P2 where folks can read more about it if they'd like: pMz3w-ix0-p2

Reply:
Thank you!